core/tests/puppeteer-acceptance-tests/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -17,7 +17,7 @@
  * tag properties from the blog-admin page.
  */
 
-/** TODO(#16552): One user-story left when user cannot publish a blog post if
+/** TODO(#16552): Write a test when user is not able to publish a blog post if
```

Comment:
Write a test to check that the user is ...

if title duplicates with an --> if its tile duplicates that of an

Reply:
done.. thanks!